VirtoCommerce.Storefront/Domain/ImageProcessing/ImagekitImageProcessor.cs

```diff
@@ -0,0 +1,32 @@
+using System.Linq;
+using Microsoft.Extensions.Options;
+using VirtoCommerce.Storefront.Model.StaticContent;
+
+namespace VirtoCommerce.Storefront.Domain.ImageProcessing
+{
+    public class ImagekitImageProcessor : IImageProcessor
+    {
+        private readonly string _serviceUrl;
+
+        public ImagekitImageProcessor(IOptions<ImageProcessorOptions> options)
+        {
+            _serviceUrl = options.Value.ServiceUrl.TrimEnd('/');
```

Comment:
Value and ServiceUrl can be null. Check them.

Reply:
ok